### DISCUSSION TITLE
Supports MPG X570 Gaming Plus?

### DISCUSSION THREAD

**Discussion by rabaukin** [ixjf/MSIRGB#205] [Q&A]
Does MSIRGB support the MSI MPG X570 Gaming Plus mainboard? Can't find it in the list.

**Comment by ixjf** [comment]
Likely not. Other 550/570 motherboards don't seem to be supported.

**Reply by rabaukin** [reply]
That's a pity. :/ Mystic Light is crap without individual settings for rgb. Thanks for the quick reply :)
